site/src/docs/components/status-label/index.mdx

```diff
@@ -56,6 +56,24 @@ StatusLabel comes in five (5) different types. One neutral and four (4) with dif
   </StatusLabel>
 </PlaygroundPreview>
 
+#### Rounded
+
```

Comment:
@hannu-kivimaki Do we have any recommendations on rounded StatusLabel usage? For example when / where to use it. Or is this rounded alteration sort of a precaution for when we deprecate the rectangular tags?

Reply:
This is the latter you mentioned. A precaution before deprecating rectangular ones. The goal is to have just the rounded variants in use through all city of Helsinki services.